src/pudl/etl/epacems_assets.py

```diff
@@ -107,30 +115,36 @@ def consolidate_partitions(context, partitions: list[YearPartitions]) -> None:
         context: dagster keyword that provides access to resources and config.
         partitions: Year and state combinations in the output database.
     """
-    partitioned_path = PudlPaths().output_dir / "core_epacems__hourly_emissions"
-    monolithic_path = PudlPaths().output_dir / "core_epacems__hourly_emissions.parquet"
+    partitioned_path = _partitioned_path()
+    # Create monolithic output in base output directory and parquet directory
+    monolithic_paths = [PudlPaths().output_dir, PudlPaths().output_dir / "parquet"]
```

Comment:
Why write incrementally to both of these locations instead of copying the whole file after the fact as part of the data deployment? Parquet is very fast so I imagine this doesn't add a lot of time, but I'd imagine copying a file is faster.

With the addition of parquet outputs for all of the tables, to me it seems like the `parquet/` directory becomes the canonical parquet output location, and pulling the consolidated file up to the top level for distribution is a special case that is necessitated by our decision not to distribute all of the parquet outputs right now, and the need to have a flat file hierarchy for Zenodo and Kaggle.

Storing two copies of the consolidated file in the nightly build outputs will also increase their size by 5GB, and since we keep 30 days of build outputs, maybe increase overall storage use by ~100GB.

Reply:
That all makes sense, and I was also running the etl overnight and saw the overwrite errors, so I've gone to just writing to the `parquet/` directory. It seems like that's what we want?

Reply:
Yes, I think we can handle all the what-file-goes-where stuff in the "distribution" phase, and keep this focused on data production.